content/blog/deno/series-introduction.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Exploring the deno.land - A blog post series 🦕
+description: "Introduction to a blog post series where we use Deno for multiple contexts, from building CLIs to web scrapping tools, among others."
+date: "2021-03-23"
+published: true
+---
+
+This is the introduction for a blog post series where we'll use [Deno](https://deno.land) to build different applications going from CLIs to web scrapping tools, among others.
+
+If you're not familiar with Deno, here's the [TLDR](/deno/tldr). If you at least have heard of Deno and you are here to know more about it, let's not waste you more time.
+
+Here's what we'll cover on the upcoming posts: 
+
+- [Creating a CLI with Deno](/deno/build-a-cli-with-deno)
+- Using Deno with Puppeteer
+- Building a Deno plugin, the interoperability with Rust
+- Static site generation in Deno, with aleph.js
+- ...
+
+*Most of these posts are written in collaboration with friends working in the industry. I believe this will make such posts more interesting -- as these are people with completely different backgrounds and experiences.*
+
+We'll use these example applications to explore Deno and how it can be applied to common tasks. By the end of it, you will have a better understanding of this new JavaScript runtime and how it can be used.
+
+## Let's start!
+
+From now on it is up to you. 
+
+The above posts do not require to be read in any specific order, but if you want a suggestion, you can start by [Creating a CLI with Deno](/deno/build-a-cli-with-deno) written by my friend [Joni Oliveira]() and me.
+
+I hope you find it interesting.
+
+Best,
```

Comment:
Seems to be a bit lost "Best," not followed by a name, as it seems the short for `best regards`